### PR TITLE
{bio}[dummy] SRA-Toolkit 2.8.2-1 (+ fix name for 2.5.7 easyconfig)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2021.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c915e7a101164d3f1b143fc3c26ca6c4 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
ca001.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/110f24d30f7b5a8792cfb791c5ef33a3 for a full test report.

File: easybuild/easyconfigs/s/SRA-Toolkit/SRA-Toolkit-2.5.7-centos_linux64.eb [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 easyblock = 'PackedBinary'
 
-name = 'sratoolkit'
+name = 'SRA-Toolkit'
 version = '2.5.7'
```
Comment (verdurin) [inline easybuild/easyconfigs/s/SRA-Toolkit/SRA-Toolkit-2.5.7-centos_linux64.eb]: Travis is complaining about a discrepancy with the expected filename:
```
 Filename '/home/travis/build/easybuilders/easybuild-easyconfigs/easybuild/easyconfigs/s/SRA-Toolkit/SRA-Toolkit-2.5.7-centos_linux64.eb' of parsed easyconfig matches expected filename 'SRA-Toolkit-2.5.7.eb'
```

and

```
Filename '/home/travis/build/easybuilders/easybuild-easyconfigs/easybuild/easyconfigs/s/SRA-Toolkit/SRA-Toolkit-2.8.2-1-centos_linux64.eb' of parsed easyconfig matches expected filename 'SRA-Toolkit-2.8.2-1.eb'
```

It's expecting `centos_linux64` to be the `versionsuffix`?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2021.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/316a1a25ec1eae4092ae5b8920a30448 for a full test report.

File: easybuild/easyconfigs/s/SRA-Toolkit/SRA-Toolkit-2.5.7-centos_linux64.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/s/SRA-Toolkit/SRA-Toolkit-2.5.7-centos_linux64.eb]: Indeed, I overlooked that, fixed in 28d19f8

**Comment by boegel** [comment]
@verdurin Good to go now...

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**